commands/wiki/ores/copper_ore.js

```diff
@@ -0,0 +1,17 @@
+const { Command } = require('discord.js-commando');
+
+module.exports = class WikiLink extends Command {
+	constructor(client) {
+		super(client, {
+			name: 'copperOre',
+			aliases: ['copper'],
+			group: 'oreWiki',
```

Comment:
```suggestion
			group: 'wiki-ores',
```
Group names must be lower case and must be identical to the folder path. Please change this and the folder it's in to be commands/wiki-ores or just ores